Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <team.coretools@kisio.com>", "Guillaume Pinot <texitoi@texitoi.eu>"]
 name = "transit_model"
-version = "0.44.0"
+version = "0.45.0"
```

Comment:
No need for that, the version `0.44.0` has not yet been released.

Reply:
done